tests/TestCase/Database/WindowQueryTests.php

```diff
@@ -0,0 +1,84 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The Open Group Test Suite License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         4.1.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Test\TestCase\Database;
+
+use Cake\Database\Query;
+use Cake\Datasource\ConnectionManager;
+use Cake\TestSuite\TestCase;
+
+/**
+ * Tests WindowExpression class
+ */
+class WindowQueryTests extends TestCase
+{
+    protected $fixtures = [
+        'core.Comments',
+    ];
+
+    public $autoFixtures = false;
+
+    /**
+     * @var \Cake\Database\Connection
+     */
+    protected $connection = null;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->connection = ConnectionManager::get('test');
+
+        $skip = env('SKIP_WINDOW_TESTS');
```

Comment:
Can we put the skip condition logic into the test here? Without that logic here tests will fail when people run test locally.

Reply:
Updated the skip test logic.